src/compiler/types/validate-primary-package-output-target.ts

```diff
@@ -38,25 +49,32 @@ export const PRIMARY_PACKAGE_TARGET_CONFIGS = {
   dist: {
     getModulePath: (rootDir: string, outputTargetDir: string) =>
       normalizePath(relative(rootDir, join(outputTargetDir, 'index.js'))),
-    getTypesPath: (rootDir: string, outputTargetConfig: d.OutputTargetDist) =>
+    getTypesPath: (rootDir: string, outputTargetConfig: any) =>
```

Comment:
Any reason we have to type this as `any`?

Reply:
Yes, this was intentional. I'll add a comment